src/main/java/ch/njol/skript/entity/StriderData.java

```diff
@@ -0,0 +1,85 @@
+package ch.njol.skript.entity;
+
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.Strider;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Objects;
+
+public class StriderData extends EntityData<Strider> {
+
+	static {
+		register(StriderData.class, "strider", Strider.class, 0,
+			"warm strider", "strider", "shivering strider");
+	}
```

Comment:
```suggestion
	static {
		register(StriderData.class, "strider", Strider.class, 1,
			"warm strider", "strider", "shivering strider");
	}
```
missed this when I was working on it